### PR TITLE
Lower CPU usage

### LINKED ISSUE #199: Arrow animations are CPU intensive
Deleting the arrows significantly increase the framerate of the wind field.
Check maybe `d3.active` in https://github.com/d3/d3/blob/master/CHANGES.md#transitions-d3-transition

### PR DESCRIPTION
**Pull request by scarlac**
Arrows have been converted from SVG gradients to good old GIF images. Since the scale is static, we can generate all necessary images up front so the browser doesn't have to.
Fixes #199

### CODE REVIEW

File: web/app/exchangelayer.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ function appendGradient(element, triangleHeight) {
 }
 
 ExchangeLayer.prototype.animateGradient = function(element, color, duration) {
+    return;
```
Comment (corradio) [inline web/app/exchangelayer.js]: We should probably remove the whole method

File: web/app/exchangelayer.js [hunk 2]
```diff
@@ -82,12 +83,6 @@ function isMobile() {
     return (/android|blackberry|iemobile|ipad|iphone|ipod|opera mini|webos/i).test(navigator.userAgent);
```
Comment (corradio) [inline web/app/exchangelayer.js]: I think you can remove that method.

File: web/app/exchangelayer.js [hunk 3]
```diff
@@ -142,40 +137,37 @@ ExchangeLayer.prototype.render = function() {
...
-            return d.co2intensity > that.STROKE_CO2_THRESHOLD ? 'lightgray' : 'black';
+            var rotation = d.rotation + (d.netFlow > 0 ? 180 : 0);
+            return 'translate(' + center[0] + ',' + center[1] + '), ' + 'rotate(' + rotation + ')';
         })
         .style('display', function (d) { return (d.netFlow || 0) == 0 ? 'none' : 'block'; })
-        .select('path')
-            .transition()
-            .duration(2000)
-            .attr('transform', getTransform);
+    const arrowCarbonIntensitySliceSize = 80; // New arrow color at every X rise in co2
```
Comment (corradio) [inline web/app/exchangelayer.js]: Question: we use good old vanilla javascript. Isn't `const` an ES6 keyword? Maybe it's not supported in older browsers?

**Comment by corradio** [comment]
Also worth noting this fixed #199

File: web/app/exchangelayer.js (hunk 3 above)
Comment (scarlac) [inline web/app/exchangelayer.js]: You're right, it's an ES6 feature. It is vanilla but only supported from IE 11 and up. I'll change it.

**Pull request merged**